bindings/cloudwego/kitex/context.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kitex
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/cloudwego/kitex/client"
+	"github.com/cloudwego/kitex/client/genericclient"
+	"github.com/cloudwego/kitex/pkg/generic"
+)
+
+const (
+	metadataRPCMethodName  = "methodName"
+	metadataRPCDestService = "destService"
+	metadataRPCHostports   = "hostPorts"
+	metadataRPCVersion     = "version"
+)
+
+type kitexContext struct {
+	version     string
+	destService string
+	hostPorts   string
+	method      string
+	inited      bool
+	client      genericclient.Client
+}
+
+func newKitexContext(metadata map[string]string) *kitexContext {
+	kitexMetadata := &kitexContext{}
+	kitexMetadata.version = metadata[metadataRPCVersion]
+	kitexMetadata.destService = metadata[metadataRPCDestService]
+	kitexMetadata.hostPorts = metadata[metadataRPCHostports]
+	kitexMetadata.method = metadata[metadataRPCMethodName]
+	kitexMetadata.inited = false
+	return kitexMetadata
+}
+
+func (d *kitexContext) Init(metadata map[string]string) error {
+	if d.inited {
+		return nil
+	}
+	var destService, hostPorts string
+	destService, ok := metadata[metadataRPCDestService]
+	if !ok {
+		return fmt.Errorf("dapr bingdings cloudwego/kitex metadataRPCDestService isn't exist")
+	}
+	hostPorts, ok = metadata[metadataRPCHostports]
+	if !ok {
+		return fmt.Errorf("dapr bingdings cloudwego/kitex metadataRPCHostPorts isn't exist")
+	}
+	_, ok = metadata[metadataRPCMethodName]
+	if !ok {
+		return fmt.Errorf("dapr bingdings cloudwego/kitex metadataRPCMethodName isn't exist")
+	}
+
+	genericCli, err := genericclient.NewClient(destService, generic.BinaryThriftGeneric(), client.WithHostPorts(hostPorts))
+	if err != nil {
+		return fmt.Errorf("get gerneric service of kitex failed %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("get generic service of kitex failed %w", err)
```

What does this actually mean? Does a user of this binding know what a generic client is? And can there be any explanation why it failed?